stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -38,6 +38,22 @@ spec:
                           "mountPath": "/var/jenkins_home"
                       }
                   ]
+              },
```

Comment:
@dtzar -- I've submitted the following which allows for a better generic UX. Take a look and let me know what you think. https://github.com/kubernetes/charts/pull/916